internal/nginx/main.go

```diff
@@ -75,6 +79,8 @@ func NewPostStatusRequest(path, contentType string, data interface{}) (int, []by
 		return 0, nil, err
 	}
 
+	timeout := HealthCheckTimeout
+	socketClient := buildUnixSocketClient(timeout)
```

Comment:
Another question here, at there we will call `buildUnixSocketClient ` multiple times. Any problems here?  @aledbf

Reply:
Updated. Only initial the socket client at once.